hawkular-command-gateway-war/pom.xml

```diff
@@ -72,11 +72,6 @@
     </dependency>
 
     <dependency>
-      <groupId>org.hawkular.inventory</groupId>
-      <artifactId>hawkular-inventory-json-helper</artifactId>
-    </dependency>
```

Comment:
4db0e03 implements the idea that the request type determines routing. There are several routing Interfaces for Messages there ATM: `ResourcePathDestination`, `UiSessionDestination`, `UiSessionOrigin` and others like `GroupPathDestination` and `ResourcePathsDestination` can be added easily.